src/k2/dto/SKVRecord.h

```diff
@@ -182,7 +187,7 @@ class SKVRecord {
     // no-arg version to satisfy the template expansion above in the terminal case
     void readMany() {}
 
-    uint32_t getFieldCursor() const;
+    uint32_t getFieldCursor(CursorType cursorType) const;
```

Comment:
We should have a default argument for this parameter that is the deserializerFieldCursor so that other code, like chogori-sql, is still compatible